prov/gni/test/rdm_sr.c

```diff
@@ -1205,20 +1221,21 @@ Test(rdm_sr, send_readfrom)
 	cr_assert(rdm_sr_check_data(source, target, len), "Data mismatch");
 }
 
-void do_send_buf(void *s, void *t, int len)
+void do_send_buf(void *p, void *t, int len)
 {
 	int ret;
 	int source_done = 0, dest_done = 0;
 	struct fi_cq_tagged_entry s_cqe, d_cqe;
 	ssize_t sz;
+	uint64_t s[NUMEPS] = {0}, r[NUMEPS] = {0}, s_e[NUMEPS] = {0}, r_e[NUMEPS] = {0};
```

Comment:
It looks like a lot of the edits you made throughout push lines past the 80 character limit.  You probably want to get the linux stye checker working to point out those issues when you commit.